native/Makefile

```diff
@@ -1,24 +1,15 @@
-BENCHMARKS = fft_cdp-in-c fft_cdp-in-nc fft_cdp-out-c \
-	     fft_cdp-out-nc fft_rdp-out-c fft_rdp-out-nc \
-	     rfft_rdp-out-c rfft_rdp-out-nc \
-	     fft_rdp2-out-c fft2_cdp-in-c fft2_cdp-in-nc \
-	     fft3_cdp-in-c fft3_cdp-in-nc fft2_cdp-out-c \
-	     fft2_cdp-out-nc fft3_cdp-out-c fft3_cdp-out-nc
+SOURCES = fft_bench.c
 
 CC = icc
 CFLAGS = -m64 -fPIC -fp-model strict -O3 -g -fomit-frame-pointer \
 	 -DNDEBUG -qopenmp -xSSE4.2 -axCORE-AVX2,COMMON-AVX512 \
-	 -lmkl_rt
+	 -lmkl_rt -DDEBUG -Wall -pedantic
```

Comment:
What is the point of having `-DDEBUG` alongside `-DNDEBUG` ?

Reply:
I had written some debug messages in an earlier revision enabled with `-DDEBUG`, but they've been since removed, so this flag is now useless. I went ahead and removed it - thanks!